@ecomplus/storefront-template/template/scss/components/_header.scss

```
@@ -125,9 +125,10 @@
     }
 
     &--full {
-      width: 98%;
-      max-width: 1200px;
-      max-height: 600px;
+      width: 100%;
+      max-width: 100%;
+      max-height: 400px;
+      overflow-y: scroll;
```

```suggestion
      overflow-y: auto;
```

Só é pra ter scroll quando necessário..